### DISCUSSION TITLE
Response Curves and Contribution DF for each Geo

### DISCUSSION THREAD

**Discussion by xijianlim** [google/lightweight_mmm#29] [Ideas]
Hi, Is there a way to get response curves and contribution breakdowns on the Geo level?

Also, will the model be able to provide coefficients for the hill function and adstock after fitting?

**Comment by pabloduque0** [comment]
Hey @xijianlim ! Sorry I didnt see this one.

At the moment both response curve and contributions are given at a national level (so summing all geos).

Yess, you can find the values for the parameters of the hill and adstock functions in `print_summary` and `your_mmm.trace`.

**Comment by stdp95** [comment]
@pabloduque0  I was just wondering when we can expect to receive the response curves and media metrics at the geo level.

**Reply by theslothook** [reply]
@xijianlim @stdp95 I have adapted their code to get the response curve breakdowns by the geo level. It amounts to not summing the matrices by geo
